KBH/4Day/완주하지 못한 선수.js

```diff
@@ -0,0 +1,12 @@
+function solution(participant, completion) {
+    let answer = '';
+    const sortParti = participant.sort();
+    const sortComple = completion.sort();
+    for(let i = 0; i < participant.length; i++){
+        if(sortParti[i] !== sortComple[i]){
+            answer = sortParti[i];
+            break;
+        }
+    }
+    return answer;
+}
```

Comment:
```suggestion
function solution(participant, completion) {
    let answer = '';
    const sortParti = participant.sort();
    const sortComple = completion.sort();
    
    for (let i = 0; i < participant.length; i++) {
        if (sortParti[i] !== sortComple[i]) {
            answer = sortParti[i];
            break;
        }
    }
    return answer;
}
```

변수 선언부와 for문, while문 if문 등을 문맥에 따라 줄바꿈으로 나눠주는게 가독성에 좋습니다~ 꿀팁

Reply:
@KimBYH sort에 대해 @Ryu-02 명규님께 피드백드린게 있는데 아래 코멘트 꼭 읽어주세요
https://github.com/modern-agile-team/9term-assignment/pull/28#discussion_r1873250608